### DISCUSSION TITLE
40x oil vs 20x air, working distances and chamber bottom thickness

### DISCUSSION THREAD

**Discussion by seiryoku-zenyo** [IBEXImagingCommunity/ibex_imaging_knowledge_base#210] [General]
Hi all,

Our lab is now planning to setup an IBEX platform. First of all, thanks for the initiative of building this community!

1- Some of the concerns we have out of the bat relate to the amount of oil needed to scan through large tissue sections. With so many continuous cycles, what is your experience in making sure that there is enough oil to avoid dry contact between the surfaces, but without going overboard and having oil running down the objective? Could lightly pre-oiling the chamber bottom be an option?

2- For some large sections like ours, a 40x objective, as in the original paper, would produce a gigantic mosaic of sometimes about 1000 tiles. In such cases, if we decide to use a 20x objective, would the working distances be compatible with the recommended chambers and their bottom thicknesses? Anyone with IBEX experience using 20x?

3- Finally, during the Fluigent webinar, I heard Dr. Chu commenting about the discontinuation of the recommended chamber. Any updates on trials using different chambers and their specs?

Kind regards,

**Comment by zivy** [comment]
Hello @seiryoku-zenyo,

Can only answer your third question, the two others are outside my wheelhouse. 

Thanks to other members of the community, Ed Allen and Ryan Moriarty, there are design and build instructions for the 3D chamber, so you can print your own. For details, see [the announcement](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/discussions/192) we made a while back.

**Reply by seiryoku-zenyo** [reply]
Thank you very much for the information, @zivy!

One must love open sourceness!

all the best

**Comment by radtkea** [comment]
Hi @seiryoku-zenyo, 

Welcome to the community! We are so glad you are here and asking questions in the open!

1. This is an excellent question! Applying the right amount of oil is a sacred art. In our [Nature Protocols](https://www.nature.com/articles/s41596-021-00644-9#Sec29), we state, _"coat the bottom surface of the coverslip generously with immersion oil across a wide region and avoid bubbles."_ Easy to say. Hard to do. **Yes, I recommend that you pre-oil the chamber bottom.** For manual IBEX I do the following between each imaging cycle: 
- Remove sample from microscope stage.
- Use lens paper to remove excess oil from the bottom of the chamber.
- Dye inactivate fluorophores.
- Perform antibody labeling.
- Mount samples with Fluoromount-G.
- Important step before placing back on stage: Clean bottom of chamber with lens cleaner and lens paper.
- Apply generous drop of immersion oil.
- Carefully inspect immersion oil on chamber bottom for any bubbles. Remove with a P10 before placing on stage. **(See picture 1)**.
- **Ice the cake:** This is my very technical term (lol) for evenly distributing the oil across the chamber bottom. I do this by doing a spiral scan in the Navigator mode of our Leica instruments. You could also do this by moving the objective around the stage with your x-y controller, manually.
- Avoid large/irregular shapes when creating a region of interest. This stretches the oil and often results in weird artifacts **(See picture 2).**

Picture 1: Air bubble in immersion oil creates a dark shadow that moves around the image. Solution: Carefully inspect pre-oiled chamber bottom before placing on stage. Remove with a pipette. These steps need to be repeated before imaging each cycle.
![Foil_Oil_2a](https://github.com/user-attachments/assets/856a694f-9e2d-414a-8c78-8c2f10bfd48a)

Picture 2: Insufficient oil will appear as "dark, out of focus areas". You see that the first 5 rows are in focus, bright yellow cells. There is a progressive decline in signal and then the entire image goes black. This is not a stage tilt. It's insufficient oil covering the bottom half of the specimen. Solution: Generous application of oil + "icing the cake" before imaging.
![Foil_Oil_1a](https://github.com/user-attachments/assets/53544b5a-4a40-4351-9875-19a97bf93de6)

2. Wow, 1,000 tiles would be hard to manage for a number of reasons. We tried to keep our 40X tile scans under 200 tiles. We used a 40X (NA 1.3) objective because we needed to resolve densely packed cells. One member of the IBEX Imaging Community used a 20X objective (Dry, 0.8 NA) and a Leica Thunder to obtain whole slide images of human FFPE lung tissue sections. Aleksandra Lunich in the Anidi lab is writing a JoVE article for the [IBEX collection](https://app.jove.com/methods-collections/3182). It always comes down to your biological question(s), imaging needs, time of image acquisition, and size of the output files. I always experiment with image acquisition parameters before embarking on a new imaging study. For example, you can create an image with a 40X oil objective versus a 20X air objective and subject both to downstream segmentation. You can experiment with collecting different sized regions of interest. Perhaps, you don't need to scan the entire tissue section, but instead can go to select ROIs. The working distance of the 20X and 40X is not an issue. 

3. Yes, we are so excited about this contribution from the community!

Best of luck! We're here to help.

**Reply by seiryoku-zenyo** [reply]
Many thanks for the informative reply, @radtkea!

We were considering an approach using something like Aria together with a TTL interface towards something like Thunder. So far we only have experience with traditional cycif using a 20x.

Using the 40x for ROIs as opposed to WSIs sounds about right. 

All the best!
